tests/components/wallbox/__init__.py

```diff
@@ -52,6 +52,30 @@
     )
 )
 
+test_response_bidir = json.loads(
+    json.dumps(
+        {
```

Comment:
What's the reason for dumping and immeditaly loading it again?

```suggestion
test_response_bidir = {
```

Reply:
I am guessing a copy, since this is how you do that in javascript

Reply:
I use this to load a nested json result quickly. Might be an ugly solution, but its the quickest for me and gives me valid datatypes in the result. If you have a good alternative I could change it (maybe seperate PR, I use this i more tests)

Reply:
Quick update, I just tried it without both the load and the dumps -> works flawlessly. Don't know, probably had an issue before which this fixed. I suggest removing the loads/dumps construct from this new part in this PR. I will make a seperate PR to clean up the other tests.